silkworm/silkrpc/http/connection.hpp

```diff
@@ -50,6 +50,7 @@ class Connection {
     Connection(boost::asio::io_context& io_context,
                commands::RpcApi& api,
                commands::RpcApiTable& handler_table,
+               std::vector<std::string>& allowed_origins,
```

Comment:
I suggest changing this to a `const` reference to clearly communicate to the reader and also enforce that `Connection` won't change the allowed origins